### DISCUSSION TITLE
Update render every other frame?

### DISCUSSION THREAD

**Discussion by zeevo** [remotion-dev/remotion#907] [General]
Hello

I'd like to simulate a slowdown, and just be able to add different speeds to my `Sequence`s where applicable. Is there a way to _slow down_ renders in some way?

I have tried rendering every odd frame, but I run into our favorite "React: Too many re-renders" error as I do not think `useCurrentFrame()` updates every time a component updates... and things start to get complicated fast.

Has anyone been able to implement _slow-downs_ or things such as that while maintaining the same FPS?

**Comment by JonnyBurger** [comment]
Hi @zeevo, interesting question! 

My idea would be to just multiply the return value of `useCurrentFrame`:

```ts
const frame = Math.round(useCurrentFrame() * 0.5)
```

and drive all your animations based off that. This is similar in style to how the playback speed feature works in our editor.
